### ISSUE TITLE
Wrong output in example

### ISSUE THREAD

**Issue by okachynskyy** [jakubroztocil/rrule#381]
- "rrule": "^2.6.3"
- macOS Mojave
- $ date -> Wed Dec 18 15:01:44 EET 2019

Seems like in the section about Timezone wrong output example
https://github.com/jakubroztocil/rrule#timezone-support

```
// RIGHT: Will produce dates with recurrences at the correct time
new RRule({
  freq: RRule.MONTHLY,
  dtstart: new Date(Date.UTC(2018, 1, 1, 10, 30)),
  until: new Date(Date.UTC(2018, 2, 31))
}).all()

[ '2018-02-01T10:30:00.000Z' ]
```

But after local testing I receive two records in the array:
`[ 2018-02-01T10:30:00.000Z, 2018-03-01T10:30:00.000Z ]`

**Comment by davidgoli** [comment]
You are correct, the example is wrong. The intention was to draw attention to the generated times, but this should be fixed. Good catch!

**Comment by davidgoli** [comment]
Fixed via 55be5f6
